app/controllers/home_controller.rb

```diff
@@ -1,4 +1,11 @@
+# frozen_string_literal: true
+
 class HomeController < ApplicationController
+  before_action :require_login
   def index
+    unless current_user.status
+      sign_out
+      redirect_to sign_in_path
+    end
```

Comment:
Delete the `before_action :require_login` of HomeController. Since the purpose is that the user can access without an account. Do I recommend that you use the .present? 

![imagen](https://user-images.githubusercontent.com/12243331/107106663-41e1dd80-67f2-11eb-9f5b-a265e42a5533.png)

In this way you solve the error of not finding the current_user.